pkg/controllers/core/config.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package core
+
+const (
+	kustomizePrefix = "riff-core" // kustomize adds this prefix to all our resource names
+
+	coreDeployerConfigMapName = kustomizePrefix + "-default-domain"
```

Comment:
Let's use a single ConfigMap for all config rather than a ConfigMap for each key.

```suggestion
	configMapName = kustomizePrefix
```

Reply:
I've updated to use a more generic name. The config map generated by kustomize is always prefixed with `riff-core-`. The config is now named `riff-core-settings`.